src/errno.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Now it's suggested to dual license for Apache-2.0 and MIT. Please help to add support of MIT.

Reply:
> Now it's suggested to dual license for Apache-2.0 and MIT. Please help to add support of MIT.

@jiangliu  I just updated the codes regarding to the license and other review comments. 
BTW, for the vhost utility, could you explain your request in details so I can help?